capa/render/vverbose.py

```diff
@@ -140,23 +144,28 @@ def render_feature(ostream, match: rd.Match, feature: frzf.Feature, indent=0):
     if value is None:
         raise ValueError("%s contains None" % key)
 
-    if key not in ("regex", "substring"):
+    if not isinstance(feature, (frzf.RegexFeature, frzf.SubstringFeature)):
         # like:
         #   number: 10 = SOME_CONSTANT @ 0x401000
-        if key == "string":
+        if isinstance(feature, frzf.StringFeature):
             value = render_string_value(value)
 
-        if key == "number":
+        elif isinstance(
+            feature, (frzf.NumberFeature, frzf.OffsetFeature, frzf.OperandNumberFeature, frzf.OperandOffsetFeature)
+        ):
             assert isinstance(value, int)
-            value = hex(value)
+            value = f"0x{value:X}"
```

Comment:
is this better vs. our uniform hex helper?

Reply:
<img width="172" alt="image" src="https://user-images.githubusercontent.com/156560/206199652-c651f4f2-da17-45a5-b650-abb6509cd438.png">
we have four different implementations...